Basic/Pod/FAQ.pod

```diff
@@ -367,30 +367,34 @@ pages).
 
 =back
 
-If you are interested in PDL in general you can join the PDL mailing
-list pdl-general@lists.sourceforge.net. This is a forum to discuss programming
-issues in PDL, report bugs, seek assistance with PDL related problems,
-etc. To subscribe, fill out the form at
-L<http://lists.sourceforge.net/lists/listinfo/pdl-general> .
-
-A searchable archive and a hypertext version of the traffic on this
-list (1997-2004) can be found at
-L<http://www.xray.mpe.mpg.de/mailing-lists/perldl/> . More recent
-messages (since June 2005) can be found at TBD.
+If you are interested in PDL in general you can join the pdl-general mailing
+list. This is a forum to discuss programming issues in PDL, report bugs, seek
+assistance with PDL related problems, etc.
 
 If you are interested in all the technical details of the ongoing PDL
-development you can join the PDL developers mailing list
-pdl-devel@lists.sourceforge.net . To subscribe, fill out the form at
-L<http://lists.sourceforge.net/lists/listinfo/pdl-devel> .
+development you can join the pdl-devel mailing list.
 
-A searchable archive and a hypertext version
-of the traffic on this list (1997-2004) can be found at 
-L<http://www.xray.mpe.mpg.de/mailing-lists/pdl-porters/> . More recent messages 
-(since June 2005) can be found at TBD
+Subscription and current archive links to both mailing lists can be
+found at L<http://pdl.perl.org/?page=mailing-lists>.
 
 Cross-posting between these lists should be avoided unless there is a
 I<very> good reason for doing that.
 
+The PDL project, begun in the late 1990s, has undergone considerable evolution
+since that time, and the support for it has as well. Thus mailing-list
+archives are in several places.  Originally pdl-general was called 'perldl',
+and pdl-devel was called 'pdl-porters'.
+
+|Time Period | URL                                                   |
+|------------|-------------------------------------------------------|
+|1996 - 2004 | http://www.xray.mpe.mpg.de/mailing-lists/perldl/      |
+|1997 - 2004 | http://www.xray.mpe.mpg.de/mailing-lists/pdl-porters/ |
+|2005 - 2015 | http://perldl.jach.hawaii.narkive.com/                |
+|2005 - 2015 | http://pdl-porters.jach.hawaii.narkive.com/           |
+|2015 -      | https://sourceforge.net/p/pdl/mailman/pdl-general/    |
+|2015 -      | https://sourceforge.net/p/pdl/mailman/pdl-devel/      |
```

Comment:
Very nice. This would be good to have on the website as well!

Reply:
On my wishlist is a better mailing list archive with searching.  I have the files from the "early days" but don't have a solution for the mail database/searching.